SAP S/4HANA SOAP 2.0 Release (#691)
mulesoft/docs-connectors: modules/ROOT/pages/sap/sap-s4hana-cloud-connector.adoc

* Put placeholder values in the `mule-artifact.content` configuration file in the `src/main/resources` 
folder of your project.
* You can either enter your credentials in the global configuration content, or 
reference a configuration file that contains these values. +

> Using a configuration file is recommended for simpler maintenance and better reuse of your project. Keeping these values in a separate file is useful if you 

Reviewer: ```suggestion
Use a configuration file to achieve simpler maintenance and better reuse of your project. Keeping these values in a separate file is useful if you 
```
